package.json

```diff
@@ -14,9 +14,9 @@
     "update": "npx npm-check -u",
     "prepublishOnly": "npm run build",
     "pretest": "npx -p \"@fastify/secure-session\" secure-session > secret-key",
-    "test": "tap",
+    "test": "borp --coverage",
     "test:report": "npm run test | tee out.tap",
-    "test:reporter": "tap-mocha-reporter xunit < out.tap > junit-testresults.xml",
+    "test:reporter": "npm run test --reporter=junit>junit-testresults.xml",
```

Comment:
where is this used?

Reply:
It's not being used in the existing code.